### ISSUE TITLE
automatic lr schedule

### ISSUE THREAD

**Issue by fcakyon** [open-mmlab/mmyolo#298]
### What is the problem this feature will solve?

The original YOLOv5 repo calculates the learning rate and schedule automatically based on the given epoch number, batch size, and GPU count.

In MMYOLO, default lr is 0.01, but I assume it is calculated for 8 batch size x 8 GPU.

It is a manual effort to scale the lr in fine-tuning task when using N GPUs with M batch size per GPU.

### What is the feature you are proposing to solve the problem?

Automatically calculating the lr using batch size and GPU count information.

**Comment by hhaAndroid** [comment]
@fcakyon Thank you. We will integrate this feature as soon as possible.
